### PR TITLE
Update node operator guide to reference simple-celo-node

### PR DESCRIPTION
**Pull request by piersy**
The node operator docs have become much simpler, because the [simple-celo-node](https://github.com/celo-org/simple-celo-node)
project is now encapsulating a lot of that complexity.

### CODE REVIEW

File: docs/cel2/l2-operator-guide.md [hunk 1]
```diff
@@ -1,437 +1,30 @@
...
-:::
-
-Ensure the full migration script completes successfully (this should be clear from the logs). If it does not, please reach out for assistance.
-
-:::warning
-Please check in your migration logs that the resulting migration block must look like this for Alfajores:
-`INFO [09-26|08:28:40.753] Wrote CeL2 migration block               height=26,384,000 root=0x38e32589e0300c46a5b98b037ea85abc5b28e3a592c36ce28d07efca8983283b hash=0xe96cb39b59ebe02553e47424e7f57dbfbffca905c3ff350765985289754a00a3 timestamp=1,727,339,320
-`
+* Alfajores *26384000*
+* Baklava *not yet decided*
```
Comment (palango) [inline docs/cel2/l2-operator-guide.md:24]: This is 27110000

Comment (palango) [inline docs/cel2/l2-operator-guide.md:24]:
```suggestion
* Baklava * 27110000*
```

**Comment by lvpeschke** [comment]
Great to see it's so much shorter now!

Questions:
- is EigenDA fully covered? It's not mentioned any more, nor in https://github.com/celo-org/simple-celo-node 
- what about historical state & execution? the previous setup required running a proxy to an L1, what's the new approach?

Things I'd add:
- "node operators must upgrade their existing L1 nodes to the latest release" is too vague -> point to a specific one for Alfajores and Baklava

File: docs/cel2/l2-operator-guide.md [hunk 2]
```diff
@@ -1,437 +1,30 @@
 ---
-title: Celo L2 Migration Guide
-description: How to migrate Celo nodes from L1 to L2
+title: Celo L2 node operator guide
```
Comment (lvpeschke) [inline docs/cel2/l2-operator-guide.md:2]:
```suggestion
title: Celo L2 Node Operator Guide
```

Comment (lvpeschke) [reply docs/cel2/l2-operator-guide.md:2]: We don't have a consistent title style, but I've seen capital letters more often.

**Comment by piersy** [comment]
> "node operators must upgrade their existing L1 nodes to the latest release" is too vague -> point to a specific one for Alfajores and Baklava

**Comment by lvpeschke** [comment]
> > "node operators must upgrade their existing L1 nodes to the latest release" is too vague -> point to a specific one for Alfajores and Baklava
> 
> @lvpeschke We don't actually have those releases yet. So I'd suggest merging this and later we can make a release of celo-blockchain and then update the docs.

Ack. But then I'd change the wording to make it clear that specific releases are needed and that the info is coming (like the mainnet migration block) instead of referring to the latest one.

**Review by lvpeschke — APPROVED** [review]

**Pull request merged**